python/cugraph/tests/test_renumber.py

```diff
@@ -15,201 +15,327 @@
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
 def test_renumber_files_col(graph_file):
     gc.collect()
 
     M = utils.read_csv_for_nx(graph_file)
-    sources = cudf.Series(M['0'])
-    destinations = cudf.Series(M['1'])
+    sources = cudf.Series(M["0"])
+    destinations = cudf.Series(M["1"])
 
     translate = 1000
 
     gdf = cudf.DataFrame()
-    gdf['src'] = cudf.Series([x + translate for x in sources])
-    gdf['dst'] = cudf.Series([x + translate for x in destinations])
+    gdf["src"] = cudf.Series([x + translate for x in sources])
+    gdf["dst"] = cudf.Series([x + translate for x in destinations])
+
+    numbering = NumberMap()
+    numbering.from_dataframe(gdf, ["src"], ["dst"])
+
+    renumbered_df = numbering.add_vertex_id(
+        numbering.add_vertex_id(gdf, "src_id", ["src"]), "dst_id", ["dst"]
+    )
 
-    src, dst, numbering = cugraph.renumber_from_cudf(gdf, ['src'], ['dst'])
+    check_src = numbering.from_vertex_id(renumbered_df, "src_id")
+    check_dst = numbering.from_vertex_id(renumbered_df, "dst_id")
 
-    for i in range(len(gdf)):
-        assert sources[i] == (numbering['0'].iloc[src[i]] - translate)
-        assert destinations[i] == (numbering['0'].iloc[dst[i]] - translate)
+    assert check_src["src"].equals(check_src["0"])
+    assert check_dst["dst"].equals(check_dst["0"])
 
 
 # Test all combinations of default/managed and pooled/non-pooled allocation
-@pytest.mark.parametrize('graph_file', utils.DATASETS)
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
 def test_renumber_files_multi_col(graph_file):
     gc.collect()
 
     M = utils.read_csv_for_nx(graph_file)
-    sources = cudf.Series(M['0'])
-    destinations = cudf.Series(M['1'])
+    sources = cudf.Series(M["0"])
+    destinations = cudf.Series(M["1"])
+
+    translate = 1000
+
+    gdf = cudf.DataFrame()
+    gdf["src_old"] = sources
+    gdf["dst_old"] = destinations
+    gdf["src"] = sources + translate
+    gdf["dst"] = destinations + translate
+
+    numbering = NumberMap()
+    numbering.from_dataframe(gdf, ["src", "src_old"], ["dst", "dst_old"])
+
+    renumbered_df = numbering.add_vertex_id(
+        numbering.add_vertex_id(gdf, "src_id", ["src", "src_old"]),
+        "dst_id",
+        ["dst", "dst_old"],
+    )
+
+    check_src = numbering.from_vertex_id(renumbered_df, "src_id")
+    check_dst = numbering.from_vertex_id(renumbered_df, "dst_id")
+
+    assert check_src["src"].equals(check_src["0"])
+    assert check_src["src_old"].equals(check_src["1"])
+    assert check_dst["dst"].equals(check_dst["0"])
+    assert check_dst["dst_old"].equals(check_dst["1"])
+
+
+@pytest.fixture()
+def client_setup():
+    cluster = LocalCUDACluster()
+    client = Client(cluster)
+    client
+
+
+# Test all combinations of default/managed and pooled/non-pooled allocation
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+def test_opg_renumber(graph_file, client_setup):
+    gc.collect()
+
```

Comment:
Don't we need `cugraph.comms` to be initialized for muliple GPUs / nodes test?

Reply:
I'm thinking I should move the MG renumber tests into the dask directory with the other MG unit tests.

Reply:
The renumbering software doesn't actually do anything with cugraph.Comms, it is implemented entirely in cudf.